commit 9f0a3f09ec2d80fed2781b341527ae6bd5666cd8
Author: popcornmix <popcornmix@gmail.com>
Date:   Mon Apr 4 19:52:27 2016 +0100

    Revert "Revert "cpufreq: Temporarily ignore io_is_busy=1""
    
    This reverts commit c353af0f83220068c10f6593b1767576b9b6cc18.

diff --git a/drivers/cpufreq/cpufreq_ondemand.c b/drivers/cpufreq/cpufreq_ondemand.c
index 03ac6ce..99a9610 100644
--- a/drivers/cpufreq/cpufreq_ondemand.c
+++ b/drivers/cpufreq/cpufreq_ondemand.c
@@ -307,7 +307,12 @@ static ssize_t store_io_is_busy(struct dbs_data *dbs_data, const char *buf,
 	ret = sscanf(buf, "%u", &input);
 	if (ret != 1)
 		return -EINVAL;
-	od_tuners->io_is_busy = !!input;
+	// XXX temporary hack
+	if (input > 1)
+		input = 1;
+	else
+		input = 0;
+	od_tuners->io_is_busy = input;
 
 	/* we need to re-evaluate prev_cpu_idle */
 	for_each_online_cpu(j) {
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1227_9f0a3f09ec2d80fed2781b341527ae6bd5666cd8.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1227_9f0a3f09ec2d80fed2781b341527ae6bd5666cd8.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
